prac_05/word_occurences.py

```diff
@@ -0,0 +1,17 @@
+word_count = {}
+input_text = input("Text: ")
+input_text = input_text.split(' ')
```

Comment:
You don't need to add anything in the brackets, since it will split along the white spaces by default.